### ISSUE TITLE
Koro might serve as a more broad effort of internationalization of Go

### ISSUE THREAD

**Issue by psychoslave** [ChimeraCoder/koro#9]
Hello,

I just watched [GopherCon 2017: Aditya Mukerjee - Translating Go to Other (Human) Languages, and Back Again](https://www.youtube.com/watch?v=oDFerBdr2J0&app=desktop), which is about this project if I'm not mistaken. 

I also just launched the wiki research project [Internationalisation of Programming Languages](https://en.wikiversity.org/wiki/Internationalisation_of_Programming_Languages).

This project is very interesting, but not yet completely what I would be looking for, as it focus on Bengali. I would far more interested by it if it adopted an internationalisation approach, with Bengali as one of its proposed localisation. If you are interested in directing the project in that direction, I would be happy to help you. Just let me know your feeling about that, and if it is positive, how you think I might help best as a first step.

By the way, did you know that *koro* means *hearth* in Esperanto? ❤️ 

Cheers

**Comment by ChimeraCoder** [comment]
> This project is very interesting, but not yet completely what I would be looking for, as it focus on Bengali. I would far more interested by it if it adopted an internationalisation approach, with Bengali as one of its proposed localisation. If you are interested in directing the project in that direction, I would be happy to help you


Absolutely! I chose Bengali because it's a language I know, and also because it demonstrates a non-Latin script, but yes - my goal from the start was always to have multiple different languages. 


From a technical perspective, the translation steps aren't too difficult. The more challenging part is getting the people with experience in the relevant languages to start a community and process around it. Were there any languages you had in mind as as a starting point for the next steps?



> By the way, did you know that koro means hearth in Esperanto? ❤️


I didn't - that's a great coincidence, though!

**Comment by psychoslave** [comment]
> The more challenging part is getting the people with experience in the relevant languages to start a community and process around it. Were there any languages you had in mind as as a starting point for the next steps?

I could help on both Esperanto and French. Currently I'm finishing the translation of [the Lua manual reference into Esperanto](https://www.mediawiki.org/wiki/Extension:Scribunto/Lua_reference_manual/eo) and completing a list of [English/Esperanto word correspondences](https://www.mediawiki.org/wiki/User:Psychoslave/generic_glossary/eo), that I plane to turn into a glossary later.

I didn't play with Go so far, but that might be a good opportunity to start looking at it. Let me know what step I might go through to help Koro be ported to other languages. :)
